activities/supporting-codebase-governance/game/run-governance-game-workshop.md

```diff
@@ -11,26 +11,49 @@ redirect_from:
 
 ### Participants
 
-The game can be played with any number of participants. As [science shows](https://doi.org/10.1016/j.evolhumbehav.2016.05.005) that discussions in groups larger than four tend to split, aim for 2-4 participants. It is possible to run two groups in parallel so if there is 6 or more participants split them into two groups if space allows for it.
+The game can be played with any number of participants.
+As [science shows](https://doi.org/10.1016/j.evolhumbehav.2016.05.005) that discussions in groups larger than four tend to split, aim for 2-4 participants.
```

Comment:
```suggestion
Since [discussions in groups larger than four tend to split](https://doi.org/10.1016/j.evolhumbehav.2016.05.005), aim for 2-4 participants.
```